### PR TITLE
Business country translations

### PR DESCRIPTION
**Pull request by axelsson**
- Extracted all translations for climate reports into their own file, many of them were in the common file
- Changed tooltip for office heating so that it considers the picked country rather than the page language
- Changed area and distance units to consider chosen country as well

These changes moves all (afaik) country related logic for the form into the `country_switch_controller.js`.

### CODE REVIEW

**Comment by axelsson** [comment]
[Tooltip text on business calculator should depend on country selected and not language chosen.](https://trello.com/c/ngCIBum9/834-tooltip-text-on-business-calculator-should-depend-on-country-selected-and-not-language-chosen)

**Review by kallenilver — APPROVED** [review]
Looks good!

File: app/controllers/business/climate_reports_controller.rb [hunk 1]
```diff
@@ -37,6 +37,8 @@ def new
       @report = ClimateReport.new(calculation_period: (Time.now.year - 1),
                                   calculation_period_length: 'year')
 
+      @translations = YAML.load_file("config/locales/views/business/climate_reports.#{I18n.locale[0..]}.yml")[I18n.locale[0..]]['views']['business']['climate_reports'].to_json
```
Comment (nanditagoclimate) [inline app/controllers/business/climate_reports_controller.rb:40]: 💯

**Review by nanditagoclimate — APPROVED** [review]
Great job! 🎉

**Pull request merged**